tests/integration.rs

```diff
@@ -249,15 +249,17 @@ fn tool_embed_jpeg_with_ingredients_report() -> Result<(), Box<dyn Error>> {
         .arg("-f")
         .assert()
         .success()
-        .stdout(predicate::str::contains("ingredients.jpg"))
-        .stdout(predicate::str::contains("test ingredient"))
-        .stdout(predicate::str::contains("earth_apollo17.jpg"));
+        .stdout(str::contains("ingredients.jpg"))
+        .stdout(str::contains("test ingredient"))
+        .stdout(str::contains("earth_apollo17.jpg"));
     Ok(())
 }
 
 #[test]
 fn tool_extensions_do_not_match() -> Result<(), Box<dyn Error>> {
     let path = temp_path("./foo.png");
+    println!("Hello! {:?}", path);
```

Comment:
I am thinking that this is no necessary.

Reply:
oof, thanks!